### PR TITLE
chore: correct Sbanken typography docs

### PR DESCRIPTION
**Pull request by eliland**
Added documentation for Sbanken typography and fixed a few bugs and inconsistencies for the line-height docs in all themes, and involuntary markup changes caused by line breaks done by prettier.

### CODE REVIEW

**Review by tujoworker — APPROVED** [review]
Nice work 🙏 looks good!

... and as we talked about, a e2e test would be cool to get in 👍

File: packages/dnb-design-system-portal/src/docs/quickguide-designer/typography/typographic-rules.mdx [hunk 1]
```diff
@@ -14,6 +15,23 @@ The Eufemia typographic scale is as follows:
 
 _16px, 18px, 20px, 26px, 34px, 48px_
 
+</VisibilityByTheme>
+<VisibilityByTheme visible="sbanken">
+## In general
+
+Sbanken has two fonts in its profile; Roboto and Maisson Neue. The latter is used mainly for headlines,
```
Comment (snorrekim) [inline packages/dnb-design-system-portal/src/docs/quickguide-designer/typography/typographic-rules.mdx:22]: I believe this is spelled wrong, "Maison"

**Review by snorrekim — COMMENTED** [review]
Just a small spelling mistake

**Review by snorrekim — APPROVED** [review]

File: packages/dnb-design-system-portal/src/docs/quickguide-designer/typography/typographic-rules.mdx (hunk 1 above)
Comment (eliland) [reply packages/dnb-design-system-portal/src/docs/quickguide-designer/typography/typographic-rules.mdx]: I have misspelled this so many places! Search and replace is my friend.

**Comment by eliland** [comment]
Added cypress tests to catch some previously seen issues (line-heights wrong due to changed line-height-basis, p elements inside headlines, ++), and added a few others. We might want to consider adding tests for CSS that is specific for the ui theme in the future.

As a side note @tujoworker, the themeSwitch and pageNavigation tests always failed for me. Could you check if you see the same issue on your end? Suspect there might be some outdated tests there, but I didn't dig too deep.

**Review by tujoworker — APPROVED** [review]
Nice work!

**Pull request merged**